src/PythonExternalLinter.php

```diff
@@ -61,6 +76,13 @@ public function setLinterConfigurationValue($key, $value) {
       $this->getProjectRoot());
   }
 
+  public function getInstallInstructions() {
+    if ($this->customInstallInstructions) {
+      return pht('Install %s using `%s`.', $this->getPythonBinary(), $this->customInstallInstructions);
+    }
+    return pht('Install %s using `pip install %s`.', $this->getPythonBinary(), $this->getPipPackageName());
```

Comment:
We're losing a little nuance here around `pip` versus `pip3` for Python 3-specific packages. I don't think it's worth adding a bunch more code for that, however, and users can always use `install-instructions` to provide a more specific message.

Reply:
Extending the `checkVersion` pattern that's in `Flake8Linter` seems like a better way to handle Python 3-specific packages.